packages/demo/src/App.svelte

```diff
@@ -16,6 +16,7 @@
   import gas from '@web3-onboard/gas'
   import dcentModule from '@web3-onboard/dcent'
   import sequenceModule from '@web3-onboard/sequence'
+  import tallyHoModule from '@web3-onboard/tallyho'
```

Comment:
For this to be added here the TallyHo package will need to be added to the Demo package's package.json file